### PR TITLE
Include assignment URL

### PR DESCRIPTION
**Pull request by stacksjb**
Add the URL to the assignment, formatted in markdown so that the todoist task is clickable.

Not done: Modifications to allow for different canvas URLs.

### CODE REVIEW

**Comment by stacksjb** [comment]
Added 401 errors, regex numbers.

File: easy_run.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def initialize_api():
         f.writelines(keys)
         f.close()
     else:
-        if keys[0] == "Replace THIS line with the Todoist API token. Remove trailing spaces\n" :
+        if keys[0] == "Replace THIS liane with the Todoist API token. Remove trailing spaces\n" :
```
Comment (scottquach) [inline easy_run.py]: Should stay as "line"

**Review by scottquach — CHANGES_REQUESTED** [review]

File: easy_run.py (hunk 1 above)
Comment (stacksjb) [reply easy_run.py]: Thanks, good catch. Appreciate the help!

**Comment by stacksjb** [comment]
Updated!

**Review by scottquach — APPROVED** [review]

**Comment by stacksjb** [comment]
Thx!

**Comment by stacksjb** [comment]
Did I do something wrong? Don't show this committed yet.

**Comment by stacksjb** [comment]
Waiting for merge.

**Comment by scottquach** [comment]
Do you have an option to merge the request or just close?

**Comment by stacksjb** [comment]
Just close - says that I don't have write access.

**Pull request merged**